dvc/render/match.py

```diff
@@ -10,8 +10,12 @@
 
 
 def get_files(plots_data: Dict) -> List:
-    values = dpath.util.values(plots_data, ["*", "*"])
-    return sorted({key for submap in values for key in submap.keys()})
```

Comment:
When a error happens during collection, `values` would be:

```
{"error": ResolveError}
```
And the removed code would fail with:

```
AttributeError: 'ResolveError' object has no attribute 'keys'
```